[Healing Trinket] Eonar's compassion

Dyspho opened the pull request:
Added the antorus pantheon healing trinket (eonar's compassion).

logs:
https://www.warcraftlogs.com/reports/thQVp9F4GM8qryAR
any fight on character charite
the pantheon buff is only used in Varimathras wipe 3 fight.

The trinket icon is currently not displaying.
http://blzmedia-a.akamaihd.net/wow/icons/56/inv_antorus_green.jpg returns a 404 error
hope this should be patched in 7.3.5
(btw https://render-us.worldofwarcraft.com/icons/56/inv_antorus_green.jpg works fine)

kfinch commented on src/Parser/Core/Modules/Items/EonarsCompassion.js:
```diff
@@ -0,0 +1,84 @@
+            This did ${formatNumber(this.trinketHealing)} healing and was ${trinketPercentHPS}% of your total healing, provinding ${trinketHPS} HPS.<br><br>
+            The pantheon effect procced ${this.pantheonProc} times with an average of ${pantheonPpm.toFixed(2)} PPM.<br>
+            You casted ${this.pantheonShieldCast} shields out of the ${possiblesShield} possibles.<br>
+            This did ${formatNumber(this.pantheonShieldHealing)} absorb and was ${pantheonPercentHPS}% of your total healing, provinding ${pantheonHPS} HPS.`}>
```
It's my opinion that this tooltip is more wordy than it needs to be, though it's up to you if you want to change it. Consider also using bullet points with `<ul>` and `<li>` instead of lines with `<br>`

kfinch commented on src/Parser/Core/Modules/Items/EonarsCompassion.js:
```diff
@@ -0,0 +1,84 @@
+  pantheonShieldCast = 0;
+
+  triggerBuffs = [
+    SPELLS.EONARS_COMPASSION_PANTHEONBUFF_RDROOD.id,
```
Make it `EONARS_COMPASSION_PANTHEONBUFF_RDRUID`, leaving the 'colloquial' spelling is asking for an error somewhere down the line.

kfinch commented on src/Parser/Core/Modules/Items/EonarsCompassion.js:
```diff
@@ -0,0 +1,84 @@
+  on_byPlayer_absorbed(event) {
+    const spellId = event.ability.guid;
+    if (spellId === SPELLS.EONARS_COMPASSION_PANTHEONSHIELD.id) {
+      this.pantheonShieldHealing += (event.amount || 0) + (event.absorbed || 0);
```
I don't think absorbed events ever have an `absorbed` field, though in this case you've wrapped it with an `|| 0` so it doesn't matter much

Dyspho commented:
- modified RDROOD to RDRUID for more clarity.
- removed the absorb part on absorb event, as it was useless.
- i didn't modified the tooltip as i think it's fine this way.
